middleware/validateContributions.js

```diff
@@ -0,0 +1,19 @@
+import Joi from "joi";
+
+const contributionsValidations = {
+  validateContribute(req, res, next) {
+    const schema = {
+      amount: Joi.number()
```

Comment:
maybe we should make the amount as required what do you think

Reply:
You are right.